### PR TITLE
Feature/11 implement base http service

### PR DESCRIPTION
**Pull request by Firto**

### CODE REVIEW

File: BobManager.API/BobManager.API/Controllers/ValuesController.cs [hunk 1]
```diff
@@ -11,11 +13,17 @@ namespace BobManager.API.Controllers
...
+        }
+
         // GET api/values
         [HttpGet]
-        public ActionResult<IEnumerable<string>> Get()
+        public ActionResult<ResultDto> Get()
         {
-            return new string[] { "value1", "value2" };
+            throw new ArgumentNullException();
+            return clientErrorManager.MapErrorIDToResultDto(203, new { Ahuenno = "Da zayebic"}); // Example using client error manager
```
Comment (NerevitVasyan) [inline BobManager.API/BobManager.API/Controllers/ValuesController.cs]: Remove this

**Review by NerevitVasyan — APPROVED** [review]
Approve

File: BobManager.API/BobManager.API/Controllers/ValuesController.cs (hunk 1 above)
Comment (Firto) [reply BobManager.API/BobManager.API/Controllers/ValuesController.cs]: Resolved

**Review by ander21 — APPROVED** [review]
Approve

**Review by viviviviviivilll — APPROVED** [review]
Approve

**Pull request merged**